standard/links/commons/links.lua

```diff
@@ -90,6 +90,7 @@ local PREFIXES = {
 	},
 	['faceit-c'] = {'https://www.faceit.com/en/championship/'},
 	['faceit-hub'] = {'https://www.faceit.com/en/hub/'},
+	['faceit-l'] = {'https://www.faceit.com/en//league//a14b8616-45b9-4581-8637-4dfd0b5f6af8/'},
```

Comment:
This seems to be the link to a specific series (ESEA), not a general link

Reply:
At that point, maybe an empty prefix is better instead? and just paste the full link

Reply:
Yeah I get it, but I think you agree it's kinda wierd to add it for a specific series only